src/content/docs/features/upload.mdx

```diff
@@ -3,12 +3,87 @@ title: Upload
 description: File uploads through HTTP.
 ---
 
-import Stub from '@components/Stub.astro';
 import PluginLinks from '@components/PluginLinks.astro';
+import { Tabs, TabItem, Steps } from '@astrojs/starlight/components';
+import CommandTabs from '@components/CommandTabs.astro';
 
 <PluginLinks plugin="upload" />
 
-<Stub>
-	Based on
-	https://github.com/tauri-apps/plugins-workspace/tree/v2/plugins/upload
-</Stub>
+Upload files from disk to a remote server over HTTP. Download files from a remote HTTP server to disk.
+
+## Setup
+
+_This plugin requires a Rust version of at least **1.64**_
```

Comment:
1.75 (we need to find a better way to specify the plugin version)

Reply:
I was thinking about adding a note e.g.

:::note
This plugin requires a Rust version of at least **1.64**
:::

Reply:
think it's too much, I may have expressed myself badly, I was referring to the fact that we would have to manually check and update versions

Reply:
If we could unify the Rust version of all the plugins would be ideal tho.

Reply:
https://github.com/tauri-apps/plugins-workspace/blob/v2/README.md?plain=1#L34

So it seems they all share the same Rust version

Reply:
At least currently they do, yes. I just added it to all readmes so it's displayed correctly on crates.io/npm. If you think it's valuable we could keep them always matched (so raise msrv for all, if required by one), and/or matching it with tauri too.